src/components/pages/create-account/__specs__/EligibilityCheck.spec.jsx

```diff
@@ -245,6 +245,39 @@ describe('eligibility check page', () => {
       const eligibilityDateOfBirthInputUpdated = wrapper.find('input[placeholder="JJ/MM/AAAA"]')
       expect(eligibilityDateOfBirthInputUpdated.prop('value')).toBe('05/03/1997')
     })
+
+    it('should not display mask if user hasnt fill dob input yet', () => {
+      // when
+      const wrapper = mount(
+        <MemoryRouter>
+          <EligibilityCheck {...props} />
+        </MemoryRouter>
+      )
+
+      // then
+      const eligibilityDateOfBirthInput = wrapper.find('input[placeholder="JJ/MM/AAAA"]')
+      expect(eligibilityDateOfBirthInput.prop('value')).toBe('')
+    })
+
+    it('should display mask once user has entered at least one number in dob input', () => {
+      // given
+      const wrapper = mount(
+        <MemoryRouter>
+          <EligibilityCheck {...props} />
+        </MemoryRouter>
+      )
+      const eligibilityDateOfBirthInput = wrapper.find('input[placeholder="JJ/MM/AAAA"]')
+
+      // when
+      act(() => {
+        eligibilityDateOfBirthInput.invoke('onChange')({ target: { value: '1' } })
+      })
+      wrapper.update()
+
+      // then
+      const eligibilityDateOfBirthInputUpdated = wrapper.find('input[placeholder="JJ/MM/AAAA"]')
```

Comment:
Pas sûr que le "eligibility" soit nécessaire, ça alourdit je trouve
```suggestion
      const updatedDateOfBirthInput = wrapper.find('input[placeholder="JJ/MM/AAAA"]')
```